app/scripts/controllers/incoming-transactions.js

```diff
@@ -227,7 +225,7 @@ export default class IncomingTransactionsController {
     if (fromBlock) {
       url += `&startBlock=${parseInt(fromBlock, 10)}`
     }
-    const response = await fetch(url)
+    const response = await fetchWithTimeout(url)
```

Comment:
Do we want to add a `try/catch` here?

Reply:
What would we do in the `catch` block in these cases?

Reply:
Log an error to Sentry so we can see if we're having transaction loading errors?  Or would that not be helpful?

Reply:
These already get logged to Sentry, by virtue of being uncaught. Uncaught errors are all logged to Sentry.